### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by petrparolek**
- polished readme
- fixed links iin readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,10 +20,10 @@ Website 🚀 <a href="https://contributte.org">contributte.org</a> | Contact 
 
 ## Usage
 
-To install latest version of `contributte/image-storage` use [Composer](https://getcomposer.com).
+To install latest version of `contributte/image-storage` use [Composer](https://getcomposer.org).
 
 ```
-composer install contributte/image-storage
+$ composer require contributte/image-storage
```
Comment (paveljanda) [inline README.md]: Could you plese remove the `$` sign? That is not a part of the command

Comment (petrparolek) [reply README.md]: what do you think @f3l1x  ?

**Review by paveljanda — CHANGES_REQUESTED** [review]
Could you plese remove the $ sign? That is not a part of the command, thanks

**Comment by petrparolek** [comment]
ping @f3l1x . It was idea from him.

**Comment by paveljanda** [comment]
@petrparolek No worries, @f3l1x will agree with me :D

**Comment by petrparolek** [comment]
I don't know Guys you tangled my head.: :D

**Pull request merged**